app/models/program.rb

```diff
@@ -15,4 +15,5 @@ class Program < ActiveRecord::Base
   }.freeze
 
   validates :facility_code, presence: true
+  validates_with ProgramValidator, on: :load_csv
```

Comment:
Why `on: :load_csv`?  Shouldn't this validation be relevant all the time?

Reply:
see comments on edu_program.rb